src/content/language/extension-types.md

```diff
@@ -446,8 +446,9 @@ It's important to be aware of this quality when using extension types.
 Always keep in mind that an extension type exists and matters at compile time,
 but gets erased _during_ compilation.
 
-An expression with the extension type `E` and the representation type `R`
-as its static type will be an object with type `R` at run time.
+For example, consider an expression `e` whose static type is the
+extension type `E`, and the representation type of `E` is `R`.
+Then, the run-time type of the value of `e` is a subtype of `R`.
 Even the type itself is erased;
 `List<E>` is exactly the same thing as `List<R>` at run time.
```

Comment:
@eernstg Does this statement still make sense after changing the previous? I think so but just checking

Reply:
Yes, that's equally useful after the change.

The fact that an expression has a certain static type `T` and the value(s) of that expression can have various types as their run-time type is one thing. Line 451-452 is about the meaning of the type itself and that's an orthogonal topic. The type itself will determine the semantics of expressions like `e is List<E>`, and in this expression the static type of `e` does not matter at all.